uptime-kuma/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0
+version: 1.0.1
```

Comment:
since I am not publishing the chart to any repository/registry I had been ignoring it so far :sweat_smile:

fine with me to bump it here, yet if we do this would be a minor bump (IMHO)